shared/fs/footer/container.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import * as I from 'immutable'
+import * as Types from '../../constants/types/fs'
+import {compose, connect, setDisplayName, type Dispatch, type TypedState} from '../../util/container'
+import Footer, {type FooterProps} from './footer'
+import * as FsGen from '../../actions/fs-gen'
+
+type StateProps = {
```

Comment:
we shouldn't need to explicitly do this anymore, flow should understand compose correctly w/o it and infer it